library/systemd/src/lib/yast2/service_widget.rb

```diff
@@ -199,6 +218,12 @@ def action_widget
       )
     end
 
+    def valid_action?(value)
```

Comment:
I would document this method too. What is a valid action? (I know :smiley:, but I would like to be it documented).

Reply:
Yep, in general will give it a little more love, wanted to discuss the approach in general, it makes sense or not :)